resources/js/Pages/Collections/Components/Articles/ArticlesGridScrollable.tsx

```diff
@@ -0,0 +1,100 @@
+import classNames from "classnames";
+import { useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { ArticleCard } from "@/Components/Articles/ArticleCard";
+import { IconButton } from "@/Components/Buttons";
+import { Heading } from "@/Components/Heading";
+
+export const ArticlesGridScrollable = ({ articles }: { articles: App.Data.Articles.ArticleData[] }): JSX.Element => {
```

Comment:
@shahin-hq mostly because the slider seemed simple enough to add css-only slider but will update, also the currently "native" slider implementation doesnt allow dragging